openfisca_core/entities.py

```diff
@@ -240,12 +134,15 @@ def filled_array(self, value, dtype = None):
             warnings.simplefilter("ignore")
             return np.full(self.count, value, dtype)
 
+    def get_variable(self, variable_name, check_existence = False):
```

Comment:
> What are the benefits of extracting this to a method? 
> This saves 30 characters on two calls to `self.simulation.tax_benefit_system.get_variable`.

Reply:
Ok got it, that's to be able to mock this method in tests

Reply:
It makes sense on its own too. `self.simulation.tax_benefit_system.get_variable ` is in technical parlance a [train wreck](http://wiki.c2.com/?TrainWreck). At some later point we'll want to have a `get_variable()` on `Simulation`, which itself delegates to `TaxBenefitSystem`. Having only one call to fix will make this easier.